base_custom_info/README.rst

```diff
@@ -26,7 +37,7 @@ expected for a given record.
 
 To define a template, you need to:
 
-* Go to *Settings > Custom Info > Templates*.
+* Go to *Custom Info > Templates*.
 * Create one.
 * Add some *Properties* to it.
```

Comment:
Fixed in https://github.com/OCA/server-tools/pull/492/commits/be58eb8e63b43e2c7b176662b920c4281002da26.